dapps/README.md

```diff
@@ -25,34 +31,34 @@
   
    The `inject.js` script will create global `web3` instance with proper provider that should be used by your dapp.
 
-1. Create `./parity-myapp/Cargo.toml` with you apps details. See example here: [parity-status Cargo.toml](https://github.com/ethcore/parity-status/blob/master/Cargo.toml).
+1. Create `./parity-ui/myapp/Cargo.toml` with you apps details. See example here: [parity-status Cargo.toml](https://github.com/ethcore/parity-ui/blob/master/status/Cargo.toml).
 
    ```bash
-   $ wget https://raw.githubusercontent.com/ethcore/parity-dapps-builtins-rs/master/Cargo.toml -O ./parity-myapp/Cargo.toml
-   $ wget https://raw.githubusercontent.com/ethcore/parity-dapps-builtins-rs/master/build.rs -O ./parity-myapp/build.rs
-   $ wget https://raw.githubusercontent.com/ethcore/parity-dapps-builtins-rs/master/src/lib.rs -O ./parity-myapp/src/lib.rs
-   $ wget https://raw.githubusercontent.com/ethcore/parity-dapps-builtins-rs/master/src/lib.rs.in -O ./parity-myapp/src/lib.rs.in
-   $ vim ./parity-myapp/Cargo.toml # Edit the details
-   $ vim ./parity-myapp/src/lib.rs.in # Edit the details
+   $ cd ./parity-ui/
+   $ cp ./home/Cargo.toml ./myapp/Cargo.toml
+   $ cp ./home/build.rs ./myapp/build.rs
+   $ cp ./home/src/lib.rs ./myapp/src/lib.rs
+   $ cp ./home/src/lib.rs.in ./myapp/src/lib.rs.in
+   # And edit the details of your app
+   $ vim ./myapp/Cargo.toml # Edit the details
+   $ vim ./myapp/src/lib.rs.in # Edit the details
    ```
 
-1. Commit the results and put it to some github repo.
+1. Commit the results.
 
    ```bash
-   $ git init && git add . && git commit -am "My first parity webapp".
+   $ git add myapp && git commit -am "My first Parity DApp".
    ```
 
-# How to include your webapp in `parity`?
-1. Edit `webapp/Cargo.toml` and add dependency to your application (it can be optional)
+# How to include your DApp in `Parity`?
```

Comment:
It's described in main readme. You first need to add the dapp to `dapps/Cargo.toml` anyway.